src/openstack_mcp_server/tools/cinder_tools.py

```diff
@@ -0,0 +1,303 @@
+        mcp.tool()(self.get_volume_details)
+        mcp.tool()(self.create_volume)
+        mcp.tool()(self.delete_volume)
+        mcp.tool()(self.extend_volume)
+        mcp.tool()(self.attach_volume_to_server)
+        mcp.tool()(self.detach_volume_from_server)
+
+    def get_cinder_volumes(self) -> list[Volume]:
+        """
+        Get the list of Cinder volumes.
+
+        :return: A list of Volume objects representing the volumes.
+        """
+        # Initialize connection
+        conn = get_openstack_conn()
+
+        # List the volumes
+        volume_list = []
+        for volume in conn.block_storage.volumes():
+            attachments = []
+            for attachment in volume.attachments or []:
+                attachments.append(VolumeAttachment(
+                    server_id=attachment.get('server_id'),
+                    device=attachment.get('device'),
+                    attachment_id=attachment.get('id')
+                ))
+            
+            volume_list.append(Volume(
+                id=volume.id,
+                name=volume.name,
+                status=volume.status,
+                size=volume.size,
+                volume_type=volume.volume_type,
+                availability_zone=volume.availability_zone,
+                created_at=str(volume.created_at) if volume.created_at else None,
+                is_bootable=volume.is_bootable,
+                is_encrypted=volume.is_encrypted,
+                description=volume.description,
+                attachments=attachments
+            ))
+
+        return volume_list
+
+    def get_volume_details(self, volume_id: str) -> Volume:
+        """
+        Get detailed information about a specific volume.
+
+        :param volume_id: The ID of the volume to get details for
+        :return: A Volume object with detailed information
+        """
+        conn = get_openstack_conn()
+        
+        volume = conn.block_storage.get_volume(volume_id)
+        
+        attachments = []
+        for attachment in volume.attachments or []:
+            attachments.append(VolumeAttachment(
+                server_id=attachment.get('server_id'),
+                device=attachment.get('device'),
+                attachment_id=attachment.get('id')
+            ))
+        
+        return Volume(
+            id=volume.id,
+            name=volume.name,
+            status=volume.status,
+            size=volume.size,
+            volume_type=volume.volume_type,
+            availability_zone=volume.availability_zone,
+            created_at=str(volume.created_at) if volume.created_at else None,
+            is_bootable=volume.is_bootable,
+            is_encrypted=volume.is_encrypted,
+            description=volume.description,
+            attachments=attachments
+        )
+
+    def create_volume(self, name: str, size: int, description: str = "", 
+                     volume_type: str = "", availability_zone: str = "") -> VolumeCreateResult:
```

Comment:
Optional 필드에 대한 정의와 초기화를 엄밀히 해주세요

```suggestion
    def create_volume(self, name: str, size: int, description: str  | None = None, 
                     volume_type: str | None = None, availability_zone: str | None = None) -> VolumeCreateResult:
```